posts/data+ai-summit-top-announcements-and-recap.md

```diff
@@ -0,0 +1,68 @@
+
+Databricks is releasing Project Lightspeed, a framework which advances Spark Structured Streaming into the real-time era as more and more new use cases and workloads migrate into streaming ([more details here](https://databricks.com/blog/2022/06/28/project-lightspeed-faster-and-simpler-stream-processing-with-apache-spark.html)).
+
+## Spark Connect
+
+Spark Connect is a simple client-server protocol that will let you run Spark anywhere! The demo on stage was done using an iPad but the possibilities are infinite (phones, embedded systems, etc).
+
+After configuring your Spark client, it will generate a query plan of your Spark code and send it to your Spark cluster so it can do the heavy work for you. You will be able to run Spark directly on your laptop without having to have a local cluster that uses all your resources.
+
+## Unity Catalog
+
+The Databricks Lakehouse Platform combines the best elements of data lakes and data warehouses to deliver the reliability, strong governance and performance of data warehouses with the openness, flexibility and machine learning support of data lakes.
+
+Unity Catalog is your unified governance solution for your Lakehouse, with key features like:
+ - Automated Data Lineage
+ - Built-in Data Search and Discovery
+ - Simplified access controls
+
+This [blog](https://databricks.com/blog/2022/06/28/whats-new-with-databricks-unity-catalog-at-the-data-ai-summit-2022.html) will give you a good overview of all the new features introduced to Unity Catalog.
+
+## MLflow 2.0
+
+Machine Learning is becoming more and more used in production and with MLflow 2.0, you will have access to MLflow pipelines to build production-grade ML pipelines.
+
+Working with ML is never easy and has its bag of challenges that can quick become a rotten tomatoes in your organization. Automating and scaling your pipelines would be pretty easy with the first release of MLflow Pipelines ([see this article](https://databricks.com/blog/2022/06/29/introducing-mlflow-pipelines-with-mlflow-2-0.html)).
+
+## Databricks Marketplace
+
+Powered by [Delta Sharing](https://databricks.com/product/delta-sharing), the Databricks Marketplace will let you discover and use a various number of data products (datasets, notebooks, dashboards, etc) from third-party vendors. In literraly few seconds, you will able to obtain a datasets of your choice and have it in your own Lakehouse.
+
+Please read [this introduction](https://databricks.com/blog/2022/06/28/introducing-databricks-marketplace-an-open-marketplace-for-all-data-and-ai-assets.html) for more details.
+
+## Databricks SQL Serverless
+
+The beauty of the Lakehouse is that you can combine Spark and SQL code so that your Data Engineers, Data Scientists and Data Analysts can use the language they're more familiar with.
```

Comment:
```suggestion
The beauty of the Lakehouse is that you can combine Spark and SQL code so that your Data Engineers, Data Scientists, and Data Analysts can use the language in which they are most familiar.
```